src/main.js

```diff
@@ -379,8 +385,33 @@ let load = loader({
   },
 
   all: {
-    requires: ['provisioner', 'server'],
-    setup: async ({provisioner, server}) => {
+    requires: ['cfg', 'provisioner', 'server', 'queue', 'WorkerType', 'monitor'],
+    setup: async ({cfg, provisioner, server, queue, WorkerType, monitor}) => {
+      if (cfg.app.publishQueueMetadata) {
+        const day = 24 * 60 * 60 * 1000;
+        let i = new Iterate({
+          maxIterationTime: day,
+          watchDog: day,
+          waitTime: day,
+          monitor,
+          handler: async (watchdog, state) => {
+            watchdog.stop();
```

Comment:
any reason for this?  The watchdog will through in this case if the handler doesn't complete in 24hours.  I suspect if this code takes 24, something is drastically wrong.

Reply:
I think that's OK -- the idea is that it doesn't really matter how much time this takes.